src/pages/LoginPage.tsx

```diff
@@ -104,6 +109,17 @@ export function LoginPage() {
 
   return (
     <IonContent>
+      <IonToolbar class="ionic-toolbar">
+        <IonRouterLink href="/">
```

Comment:
`<IonRouterLink hrer="/" >`

This isn't correct.

Change one of two.
`<IonRouterLink routerLink="/">`

or use Link component at our src/components

`<Link to="/">`

I prefer the second one.

----------------------

Additionally 

At line 143
```
<Typography sx={{ marginBottom: '18px', color: '#ff0000' }}>
            {errorMessage}{' '}
          </Typography>
          ```
          
         to 
```
<Typography sx={{ marginBottom: '18px', color: '#ff0000' }}>
            {errorMessage}
          </Typography>
          ```
          
At line 190, 200

```
{'Forgot Password?'}
```
to
```
Forgot Password?
```